compiler/luci/service/include/luci/Service/CircleNodeClone.h

```diff
@@ -31,6 +31,7 @@ void copy_common_attributes(const luci::CircleNode *src, luci::CircleNode *dst);
 
 /**
  * @brief Return a new cloned CircleNode object with same attributes value of node to graph.
+ * @note  Will return nullptr if clone has failed
```

Comment:
(optional) Maybe there's no need to describe comments with future tense.

Reply:
`will` is used for meaning of intention or design...